syntax = "proto3";

message InventoryEvent {
  string item_id = 1;
  string item_name = 2;
  int32 quantity_change = 3;
  int32 current_stock = 4;
  string location = 5;
  string event_time = 6;
}
